process/interface.go

```diff
@@ -370,12 +370,13 @@ type EndOfEpochTriggerHandler interface {
 	EpochStartRound() uint64
 	Processed()
 	Revert()
+	EndOfEpochMetaHdrHash() []byte
```

Comment:
You can delete vmcommon from imports as is reported as a redundant alias

Reply:
done